plugins/modules/azure_rm_backuppolicy.py

```diff
@@ -52,6 +52,14 @@
         choices:
             - AzureIaasVM
         type: str
+    backup_policy_type:
+        description:
+            - Defines the type of backup policy
+        default: V2
+        choices:
+            - V1
+            - V2
+        type: str
```

Comment:
Not match with argument
```suggestion
        type: str
        default: V1
```